shared/src/main/scala/squants/energy/Energy.scala

```diff
@@ -210,6 +261,16 @@ object EnergyConversions {
   lazy val btu = BritishThermalUnits(1)
   lazy val btuMultiplier = 2.930710701722222e-1
 
+  lazy val eV = ElectronVolt(1)
+  lazy val meV = MilliElectronVolt(1)
+  lazy val keV = KiloElectronVolt(1)
+  lazy val MeV = MegaElectronVolt(1)
+  lazy val GeV = GigaElectronVolt(1)
+  lazy val TeV = TeraElectronVolt(1)
+  lazy val PeV = PetaElectronVolt(1)
+  lazy val EeV = ExaElectronVolt(1)
+  lazy val evMultiplier: Double = 1.60217656535e-19
```

Comment:
Should this be moved into the `ElectronVolt` object so we don't leak a constant?